Cannot render settings submenus in FVTT v9

```
foundry.js:316 Error: An error occurred while rendering GMToolkitAdvantageSettings 48: You must specify both namespace and key portions of the setting
    at ClientSettings.get (foundry.js:2113)
    at gm-toolkit-settings.mjs:285
    at Array.forEach (<anonymous>)
    at getDataSettings (gm-toolkit-settings.mjs:272)
    at GMToolkitAdvantageSettings.getData (gm-toolkit-advantage-settings.js:18)
    at GMToolkitAdvantageSettings._render (foundry.js:3031)
    at GMToolkitAdvantageSettings._render (foundry.js:3733)
    at GMToolkitAdvantageSettings.render (foundry.js:2990)
    at SettingsConfig._onClickSubmenu (foundry.js:55267)
    at HTMLButtonElement.dispatch (jquery.min.js:2)
```

--- a/modules/gm-toolkit-settings.mjs
+++ b/modules/gm-toolkit-settings.mjs
@@ -282,6 +282,6 @@ export function getDataSettings(data, feature) {
             s.isNumber = true;
             s.inputType = "number"
         }
-        s.value = game.settings.get(s.module, s.key);
+        s.value = game.settings.get(s.namespace, s.key);
     });
 }

